packages/babel-preset-cra-universal/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "babel-preset-cra-universal",
+  "version": "1.0.0",
+  "description": "Babel preset for cra-universal",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/antonybudianto/cra-universal.git"
+  },
+  "author": "Antony Budianto, Kirin Patel",
+  "license": "MIT",
+  "files": [
+    "index.js"
+  ],
+  "dependencies": {
+    "babel-plugin-dynamic-import-node": "1.0.1",
+    "babel-plugin-syntax-dynamic-import": "6.18.0",
+    "babel-plugin-transform-class-properties": "6.23.0",
+    "babel-plugin-transform-es2015-modules-commonjs": "6.24.0",
+    "babel-plugin-transform-object-rest-spread": "6.23.0",
+    "babel-plugin-transform-react-jsx-source": "6.22.0",
+    "babel-plugin-transform-react-remove-prop-types": "0.3.2",
+    "babel-plugin-transform-runtime": "6.23.0",
+    "babel-preset-env": "1.3.3",
+    "babel-preset-react": "6.23.0"
```

Comment:
some of the package should use `@babel` scoped package, example: `@babel/preset-env`, `@babel/preset-react`